delft3dcontainermanager/tasks.py

```diff
@@ -102,3 +121,47 @@ def do_docker_sync_filesystem(self, container_id):
     """
     return False
 
+
+@shared_task(bind=True)
+def create_directory_layout(self, uuid, workingdir, parameters):
```

Comment:
All this code should be part of the do_docker_create task: the Container Model will fire only one task when the desired_State is 'created' and the docker state is 'non-existent'. The one task should first create all directories/files and then start the container.